dsl/weekly.dsl

```diff
@@ -20,7 +20,14 @@ pipelineJob('weekly_intel_testbed') {
     definition {
         cpsScm {
             scm {
-                git(ciUrl)
+                git {
+                    remote {
+                        url(vrouterUrl)
```

Comment:
Weekly job, don't need to monitor vrouter code change.
Only smoke job need to trigger by vrouter code change.

Reply:
Ok. I will modify the weekly job.